src/organisationClient.ts

```diff
@@ -36,19 +36,23 @@ export class OrganisationClient {
 
     name = name.trim().toLowerCase();
 
+    const alphabeticalOrder = () =>
+        (organisationOne: { name: string; }, organisationsTwo: { name: string; }) =>
+        organisationOne.name.localeCompare(organisationsTwo.name);
+
     return this.getOrganisations(status, address)
       .then(organisations => {
         return organisations.filter(organisation => {
           return organisation.name
             .trim()
             .toLowerCase()
             .includes(name);
-        })
+        }).sort(alphabeticalOrder())
```

Comment:
Wouldn't you get the same result by ordering in ascending order?

Reply:
Yes, This is ordering in ascended order